### ISSUE TITLE
Vi mode prompt does not show the visual mode

### ISSUE THREAD

**Issue by kubouch** [nushell/engine-q#693]
Setting `PROMPT_INDICATOR_VI_VISUAL` environment variable does not seem to have any effect. In visual mode, the prompt indicator returns to the value set in `PROMPT_INDICATOR`. The insert mode indicator works fine.

Tagging @elferherrera . This seems to happen in the reedline repo as well.

**Comment by lukexor** [comment]
I'm not sure what all situations it's meant to appear in, but I can get it to show when using the "completion" keybinding:

```
let-env PROMPT_INDICATOR_VI_VISUAL = "| "
let $config = {
  keybindings: [
    {
      name: completion
      modifier: control
      keycode: char_t
      mode: vi_insert # emacs vi_normal vi_insert
      event: { send: contextmenu }
    }
  ]
}
```
<img width="491" alt="Screen Shot 2022-01-22 at 2 30 39 PM" src="https://user-images.githubusercontent.com/709609/150657521-91825ab3-7878-48b0-aa8a-8c9790cafe9c.png">

**Comment by elferherrera** [comment]
This is because we dont have a visual mode in vi yet. In Reedline it is defined as an enum case but there is no action to get you inside it. I dont know if we should remove it  or if we are going to implement something for it.

What do you think @sholderbach  @jntrnr ?

**Comment by kubouch** [comment]
@elferherrera Ah, I was actually confusing "normal" and "visual" mode. So it makes perfect sense that setting a visual prompt indicator does not work when entering normal mode. (By "normal" mode I mean not the edit mode.)

Can we have a `PROMPT_INDICATOR_VI_NORMAL` environment variable then? I'd like to see visually whether I'm in vi or emacs mode and the vi normal mode prompt now seems to take the emacs prompt indicator.

Maybe we can also remove the visual env var if we don't use it as it might confuse people (like myself :-D).

**Comment by elferherrera** [comment]
Sure we can have that env variable. However we have been using the normal `>` as the indicator for normal mode. Im not sure if it makes sense to have another one

And I do agree that we should remove that VI_VISUAL, unless there are objections

**Comment by kubouch** [comment]
Yes please! My plan is to use `>` for emacs mode `i` for insert mode and maybe `n` for the normal mode in my prompt, When I see `>`, it somehow tells me I can type and edit, but then I realize I'm not in the insert mode. So I kind of want some mental roadblock there.

It could be set to `>` by default.

Also it's not a high priority, obviously, just a nice-to-have from my point of view.

**Comment by elferherrera** [comment]
no problem. I can add it now, since Im working with it now
